internal/templates/package.neva

```diff
@@ -0,0 +1,7 @@
+// don't forget modify i/o type and name
```

Comment:
1. looks like this template won't compile because it lacks `Main` component
2. name `package` is a little bit ambiguous. what exactly this template should be?

Reply:
This template is intended to be reference ready and is not intended to be compilable

Reply:
So what will happen if user will run `neva build --template=package`? It will create neva module without executable package? It's like for developing libraries?